index.bs

```diff
@@ -120,8 +120,12 @@ since sites can use each of the hints as a bit set on the client, and that infor
 communicated to them on every request. As such, a user agent MUST evict that
 cache whenever the user clears their cookies or when session cookies expire.
 
+Note: A site can clear the browser's `Accept-CH` cache for its origin by sending an empty `Accept-CH`
+header in a response. This corresponds to an empty |client hints set|.
```

Comment:
Maybe "This sets the origin's |client hints set} to an empty set"?

Reply:
I like it, thanks.